docs/settings.ini

```diff
@@ -14,6 +15,34 @@ java_docs = 1
 r_docs = 0
 scala_docs = 1
 
+[document_sets_1.3.1]
+clojure_docs = 1
+doxygen_docs = 1
+java_docs = 0
+r_docs = 0
+scala_docs = 1
+
+[document_sets_v1.3.1]
+clojure_docs = 1
+doxygen_docs = 1
+java_docs = 0
+r_docs = 0
+scala_docs = 1
+
+[document_sets_1.3.0]
+clojure_docs = 1
+doxygen_docs = 1
+java_docs = 0
+r_docs = 0
+scala_docs = 1
+
+[document_sets_v1.3.0]
```

Comment:
Why do we have configuration for each version twice?

Reply:
CI uses branches.

Reply:
Would it be possible to add something like this instead:
```
[document_sets_v1.3.x]
clojure_docs = 1
doxygen_docs = 1
java_docs = 0
r_docs = 0
scala_docs = 1
```